doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -902,12 +914,15 @@ and won't print anything.
 
 .. coqtop:: all
 
-   Ltac mytac tac := idtac "wow"; tac.
+   Ltac mytac tac := idtac "I am being evaluated"; tac.
 
    Goal True.
    Proof.
    Fail mytac ltac2:(fail).
```

Comment:
You could also add the corresponding example where "I am being evaluated is printed".